docs/src/index.md

```diff
@@ -3,21 +3,50 @@
     SPDX-License-Identifier: CC-BY-SA-4.0
 -->
 
-# TII SSRC Secure Technologies: Ghaf Framework
+# About Ghaf
 
 _[Ghaf Framework](./appendices/glossary.md#ghaf)_ is an open-source project that provides information about our work and studies in the security technologies field in the context of embedded virtualization.
 
 The applied software research supports _[Secure Systems Research Center](./appendices/glossary.md#ssrc)_ focus areas.
 
+*Ghaf Framework* can be used to build the *Ghaf Platform* that will provide an edge device software architecture with key features such as modularity and scalability through virtualization, support research and development of zero trust architecture (ZTA), and allow for low maintenance efforts while keeping the main code base stable and secure. The SSRC team focus is to research on enhancing ZTA to scale horizontally across edge HW platforms (phones, drones, laptops, communication modules) and vertically across SW platforms (Linux, Android, Browser, applications).
```

Comment:
We may want to introduce the SSRC acronym with the first use on line 10 like we have the "zero trust architecture (ZTA)"